.github/workflows/build_docs.yaml

```diff
@@ -53,7 +56,7 @@ jobs:
         run: pip install -v .
 
       - name: Install documenteer
-        run: pip install 'documenteer[pipelines]<0.7'
+        run: pip install 'documenteer[pipelines]<1.0'
```

Comment:
Does this need a pin at all? 0.8.3 is the latest version.

Reply:
I was playing it safe based on the comment in [DM-40059](https://jira.lsstcorp.org/browse/DM-40059).